Bindings/Java/Matlab/OpenSenseExampleFiles/IMUDataConverter.m

```diff
@@ -0,0 +1,61 @@
+%% IMUDataConverter.m
+% Example code for reading, and converting, XSENS IMU sensor data to
+% OpenSense friendly format.
+% Run this script from the OpenSenseExampleFiles directory. 
+
+% ----------------------------------------------------------------------- %
+% The OpenSim API is a toolkit for musculoskeletal modeling and           %
+% simulation. See http://opensim.stanford.edu and the NOTICE file         %
+% for more information. OpenSim is developed at Stanford University       %
+% and supported by the US National Institutes of Health (U54 GM072970,    %
+% R24 HD065690) and by DARPA through the Warrior Web program.             %
+%                                                                         %
+% Copyright (c) 2005-2019 Stanford University and the Authors             %
+% Author(s): James Dunne                                                  %
+%                                                                         %
+% Licensed under the Apache License, Version 2.0 (the "License");         %
+% you may not use this file except in compliance with the License.        %
+% You may obtain a copy of the License at                                 %
+% http://www.apache.org/licenses/LICENSE-2.0.                             %
+%                                                                         %
+% Unless required by applicable law or agreed to in writing, software     %
+% distributed under the License is distributed on an "AS IS" BASIS,       %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+% implied. See the License for the specific language governing            %
+% permissions and limitations under the License.                          %
+% ----------------------------------------------------------------------- %
+
+%% Clear any variables in the workspace
+clear all; close all; clc; 
+
+%% Import OpenSim libraries
+import org.opensim.modeling.*
+
+%% Build an Xsens Settings Object. 
+% Instantiate the Reader Settings Class
+xsensSettings = XsensDataReaderSettings('MT_012005D6_009-001_Mappings.xml');
```

Comment:
The name of the file included is imuMappings.xml
You might call it something like myImuMappings.xml to make it more clear that this can be changed.